src/radio/radio.stories.ts

```diff
@@ -12,21 +12,28 @@ storiesOf("Radio", module).addDecorator(
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-		<ibm-radio-group aria-label="radiogroup" [(ngModel)]="radio" (change)="onChange($event)">
-			<ibm-radio
-				value="radio"
-				[checked]="true">
-				zero
-			</ibm-radio>
-			<ibm-radio *ngFor="let radio of manyRadios"
-				[value]="radio.num"
-				[disabled]="radio.disabled">
-				{{radio.num}}
-			</ibm-radio>
-		</ibm-radio-group>
+		<fieldset class="bx--fieldset">
```

Comment:
thought: `fieldset` could be a component 🤔

Reply:
I mean it's only used once so I don't see a need to

Reply:
_here_ it is, but it's a pretty useful form related element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Reply:
Obviously we don't need any more change in this PR, but something to consider